client/packages/system/src/Stock/Components/Repack/RepackEditForm.tsx

```diff
@@ -13,7 +13,7 @@ import {
   StockLineFragment,
   useStock,
 } from '@openmsupply-client/system';
-import { LocationSearchInput } from 'packages/system/src/Location/Components/LocationSearchInput';
+import { LocationSearchInput } from '../../../Location/Components/LocationSearchInput';
```

Comment:
hmm, shouldn't this be just '@openmsupply-client/system' ? 
Or even just keeping the relative path, also I would have thought that relative paths (without alias) would be used to access things within the same package (makes it slightly more portable, i.e. no prerequisite root package json with aliases).

Reply:
because this file is in `system`, you'd get errors if you try to import `system` again.
what I have is a relative path - the `packages/system` is not strictly speaking relative because there is no `packages` folder in the current directory.